app/src/main/java/com/github/palFinderTeam/palfinder/profile/FirebaseProfileService.kt

```diff
@@ -101,6 +102,40 @@ open class FirebaseProfileService @Inject constructor(
         }
     }
 
+    override suspend fun muteMeetup(user: ProfileUser, meetup: String): Response<Unit> {
+        return try {
+            if (!user.canMuteMeetup(meetup)) {
+                return Response.Failure("Cannot mute this meetup.")
+            }
+            val batch = db.batch()
+            batch.update(
+                db.collection(PROFILE_COLL).document(user.uuid),
+                MUTED_MEETUPS, FieldValue.arrayUnion(meetup)
+            )
```

Comment:
You don't have to use batch for this I think, the simple update methods with arrayUnions should be equivalent